### PR TITLE
 Added code of daily Coding Exercise -11

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

**Review by amitsahu8 — APPROVED** [review]

File: First/dailyExercises/EqualityOfTwoArrays.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public class EqualityOfTwoArrays {
+    public static void main(String[] args) {
+        int[] arrayIndex = {2, 5, 1, 7, 4, 5};
+        int[] arraySubIndex = {2, 5, 1, 7, 4};
+        boolean equalOrNot = true;
+        if (arrayIndex.length == arraySubIndex.length) {
+            for (int mockData = 0; mockData < arrayIndex.length; mockData++) {
+                if (arrayIndex[mockData] != arraySubIndex[mockData]) {
+                    equalOrNot = false;
```
Comment (amitsahu8) [inline First/dailyExercises/EqualityOfTwoArrays.java]: always check for true condition

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/EqualityOfTwoArrays.java (hunk 1 above)
Comment (anushreeSonak) [reply First/dailyExercises/EqualityOfTwoArrays.java]: Done !!

Comment (amitsahu8) [reply First/dailyExercises/EqualityOfTwoArrays.java]: condition not handle properly

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/EqualityOfTwoArrays.java (hunk 1 above)
Comment (anushreeSonak) [reply First/dailyExercises/EqualityOfTwoArrays.java]: done !

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**